examples/bigquery_user_info_updater_tool/cron/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:latest
```

Comment:
jaketf on Aug 27, 2019 Member
Do you depend on ubuntu?
could use official python image python:3.6-slim or similar which will include pip and explicitly control python version.